### PR TITLE
doc/install: add manual RADOSGW install procedure

### PR DESCRIPTION
**Pull request by zdover23**
Add a manual RADOSGW installation procedure to
doc/install/manual-deployment.rst. This procedure was developed by Janne Johansson and reported to the ceph-users mailing list on 29 Jan 2024 here: https://lists.ceph.io/hyperkitty/list/ceph-users@ceph.io/message/LB3YRIKAPOHXYCW7MKLVUJPYWYRQVARU/

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/install/manual-deployment.rst [hunk 1]
```diff
@@ -461,6 +461,45 @@ In the below instructions, ``{id}`` is an arbitrary name, such as the hostname o
...
 #. Now you are ready to `create a Ceph file system`_.
 
+Manually Installing RADOSGW
+===========================
+
+For a more involved discussion of the procedure presented here, see `this
+thread on the ceph-users mailing list
+<https://lists.ceph.io/hyperkitty/list/ceph-users@ceph.io/message/LB3YRIKAPOHXYCW7MKLVUJPYWYRQVARU/>`_.
+
+#. Install RADOSGW packages on the nodes that will be the RGW nodes.
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/install/manual-deployment.rst:471]: This whole radosgw/RGW/Ceph Object Gateway ambiguity is something I'd like to address in the documentation eventually.  I've never liked that the left-screen tree pane refers to RGW as "Ceph Object Gateway". 

This is a longwinded way of saying: Accepted.

File: doc/install/manual-deployment.rst [hunk 2]
```diff
@@ -461,6 +461,45 @@ In the below instructions, ``{id}`` is an arbitrary name, such as the hostname o
...
+      .. prompt:: bash #
+
+         install -d -o ceph -g ceph /var/lib/ceph/radosgw/ceph-$(hostname -s)
+
+   b. Enter the directory just created and put the key from the earlier ``ceph
+      auth get-or-create`` command in it. Use a command similar to this command:
+
+      .. prompt:: bash #
+
+         vi /var/lib/ceph/radosgw/ceph-$(hostname -s)/keyring
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/install/manual-deployment.rst:493]: I knew it.
I previsioned it.

https://www.youtube.com/watch?v=LMT0Ki0P4mc

I'll alter this. I might use "touch" and then just instruct the reader to add the keyring to the touched file.

[the remark this answers is not in the document]
Comment (anthonyeleven) [reply doc/install/manual-deployment.rst:493]: Ack.  Pretty much everyone will always refer to it as RGW but I can see a desire for ... branding.

File: doc/install/manual-deployment.rst [hunk 3]
```diff
@@ -461,6 +461,52 @@ In the below instructions, ``{id}`` is an arbitrary name, such as the hostname o
...
+         $EDITOR /var/lib/ceph/radosgw/ceph-$(hostname -s)/keyring
+
+   c. Repeat these steps on every RGW node.
+
+#. Start the RADOSGW service by running the following command:
+
+   .. prompt:: bash #
+
+      systemctl start ceph-radosgw@$(hostname -s).service
+
```
Comment (icepic) [inline doc/install/manual-deployment.rst:510]: perhaps add "systemctl enable ceph-radosgw@$(hostname -s).service" after you see that the daemon runs, so it autostarts at next boot also.

File: doc/install/manual-deployment.rst [hunk 4]
```diff
@@ -461,6 +461,52 @@ In the below instructions, ``{id}`` is an arbitrary name, such as the hostname o
...
+         touch /var/lib/ceph/radosgw/ceph-$(hostname -s)/keyring
+
+      Use a command similar to this one to put the key from the earlier ``ceph
+      auth get-or-create`` step in the ``keyring`` file. Use your preferred
+      editor:
+
+      .. prompt:: bash #
+
+         $EDITOR /var/lib/ceph/radosgw/ceph-$(hostname -s)/keyring
+
```
Comment (icepic) [inline doc/install/manual-deployment.rst:501]: you could just concatenate the touch and $EDITOR step into one, the editor will create the file as needed.

**Pull request merged**